tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs

```diff
@@ -61,15 +62,15 @@ public void TestInitialize()
         public async Task ROPC_AAD_Async()
         {
             var labResponse = await LabUserHelper.GetDefaultUserAsync().ConfigureAwait(false);
-            await RunHappyPathTestAsync(labResponse).ConfigureAwait(false);
+            await RunHappyPathTestAsync(labResponse, authorityType: AuthorityType.Aad).ConfigureAwait(false);
```

Comment:
why does the authority have to specified?

Reply:
Added this since tenant profiles will only be populated for AAD accounts. Wanted to test that

Reply:
right, but what if a customer doesn't set it. what will happen? do we have a test for that? I would add a test for that and leave these tests as-is. if they are AAD, it should work.